### PR TITLE
Cvs clean

### PR DESCRIPTION
**Pull request by sokolov-group**
Efficient implementation of CVS-IP-ADC

### CODE REVIEW

File: pyscf/adc/dfadc.py [hunk 1]
```diff
@@ -35,6 +35,25 @@ def get_ovvv_df(myadc, Lov, Lvv, p, chnk_size):
     ovvv = ovvv.reshape(-1, nvir, nvir, nvir)
     return ovvv
 
+def get_ceee_df(myadc, L_ce, L_ee, p, chnk_size):
```
Comment (sokolov-group) [inline pyscf/adc/dfadc.py]: Redundant, remove

File: pyscf/adc/dfadc.py [hunk 2]
```diff
@@ -59,6 +78,28 @@ def get_ovvv_spin_df(myadc, Lov, Lvv, p, chnk_size):
     #vvvv = np.ascontiguousarray(vvvv.transpose(0,2,1,3)).reshape(-1, nvir, nvir * nvir)
     return ovvv
 
+def  get_ceee_spin_df(myadc, L_ce, L_ee, p, chnk_size):
```
Comment (sokolov-group) [inline pyscf/adc/dfadc.py]: Redundant, remove

File: pyscf/adc/radc.py [hunk 3]
```diff
@@ -319,6 +329,8 @@ def analyze(self):
     def compute_dyson_mo(self):
         return self._adc_es.compute_dyson_mo()
 
+    def get_opdm(self):
```
Comment (sokolov-group) [inline pyscf/adc/radc.py]: Should be consistent with PySCF convention: make_rdm1

File: pyscf/adc/radc_ip.py [hunk 4]
```diff
@@ -773,6 +773,111 @@ def compute_dyson_mo(myadc):
...
+    log = logger.Logger(adc.stdout, adc.verbose)
+
+    nroots = adc.U.shape[1]
+    U = adc.renormalize_eigenvectors(nroots)
+
+    tuple_opdm = ()
+
+    for i in range(U.shape[1]): 
+        opdm = get_opdm_eigenvectors(adc, U[:,i], U[:,i])
+        tuple_opdm += (opdm,)
```
Comment (sokolov-group) [inline pyscf/adc/radc_ip.py]: Should be list

File: pyscf/adc/radc_ip.py [hunk 5]
```diff
@@ -773,6 +773,111 @@ def compute_dyson_mo(myadc):
 
     return dyson_mo
 
+def get_opdm(adc):
```
Comment (sokolov-group) [inline pyscf/adc/radc_ip.py]: Change opdm into rdm1

File: pyscf/adc/radc_ip_cvs.py [hunk 6]
```diff
@@ -0,0 +1,1101 @@
...
+
+    # i-j block
+    # Zeroth-order terms
+
+    M_ij = lib.einsum('ij,j->ij', idn_cvs ,e_cvs)
+
+    # Second-order terms
+    t2_1 = t2[0][:]
+    t2_1_coee = t2_1[:ncvs,:,:,:].copy()
+    t2_1_ocee = t2_1[:,:ncvs,:,:].copy()
```
Comment (sokolov-group) [inline pyscf/adc/radc_ip_cvs.py:71]: Redundant amplitude

File: pyscf/adc/radc_ip_cvs.py [hunk 7]
```diff
@@ -0,0 +1,1101 @@
...
+    cput0 = log.timer_debug1("Completed M_ij second-order terms ADC(2) calculation", *cput0)
+    # Third-order terms
+
+    if (method == "adc(3)"):
+
+        eris_oovv = eris.oovv
+        eris_ovoo = eris.ovoo
+        eris_oooo = eris.oooo
+
+        eris_oecc = eris_ovoo[:,:,:ncvs,:ncvs].copy()
```
Comment (sokolov-group) [inline pyscf/adc/radc_ip_cvs.py:97]: Check for redundant integrals

File: pyscf/adc/radc_ip_cvs.py [hunk 8]
```diff
@@ -0,0 +1,1101 @@
...
+        eris_ceoc = eris_ovoo[:ncvs,:,:,:ncvs].copy()
```
Comment (sokolov-group) [inline pyscf/adc/radc_ip_cvs.py:98]: Should delete arrays after they are used if not needed

File: pyscf/adc/test/test_uadc/test_OH_uadc_ip.py [hunk 9]
```diff
@@ -35,7 +35,7 @@ def setUpModule():
     mol.symmetry = False
     mol.spin  = 1
     mol.build()
-    mf = scf.UHF(mol)
+    mf = scf.UHF(mol).density_fit()
```
Comment (sokolov-group) [inline pyscf/adc/test/test_uadc/test_OH_uadc_ip.py]: Remove

File: pyscf/adc/uadc_ao2mo.py [hunk 10]
```diff
@@ -341,6 +363,10 @@ def transform_integrals_df(myadc):
     eris.LOV = np.empty((naux,nocc_b,nvir_b))
     eris.Lvv = np.empty((naux,nvir_a,nvir_a))
     eris.LVV = np.empty((naux,nvir_b,nvir_b))
+    eris.L_ce = np.empty((naux,ncvs,nvir_a))
```
Comment (sokolov-group) [inline pyscf/adc/uadc_ao2mo.py]: Inconsistent variable naming

File: pyscf/adc/radc.py (hunk 3 above)
Comment (BigAvocado) [reply pyscf/adc/radc.py]: Renamed to get_opdm to make_rdm1

File: pyscf/adc/radc_ip.py (hunk 4 above)
Comment (BigAvocado) [reply pyscf/adc/radc_ip.py]: Converted tuple to list

File: pyscf/adc/radc_ip.py (hunk 5 above)
Comment (BigAvocado) [reply pyscf/adc/radc_ip.py]: Changed array name from opdm to rdm1

File: pyscf/adc/radc_ip_cvs.py (hunk 7 above)
Comment (BigAvocado) [reply pyscf/adc/radc_ip_cvs.py:97]: Will address this when optimizing the code. Currently, eris_ceee integrals aren't properly calculated, but rather eris_oeee is sliced to get eris_ceee

File: pyscf/adc/radc_ip_cvs.py (hunk 8 above)
Comment (BigAvocado) [reply pyscf/adc/radc_ip_cvs.py:98]: Will address when optimizing the code at a later time

**Pull request merged**